### DISCUSSION TITLE
Cupra Born

### DISCUSSION THREAD

**Discussion by cfontanella** [tillsteinbach/VWsFriend#210] [Q&A]
Hello,

Have you try with A Cupra Born? For Cupra we have another application called Cupra connect.

Thanks in advance

**Comment by tillsteinbach** [comment]
Yes, you can see the current status here: https://github.com/tillsteinbach/WeConnect-python/issues/26
It is currently too different to justify the effort for the few cars that are currently on the street. But I put my hopes on the release of SW 3.0 for all MEB cars. I hope that this will consolidate the backend a bit so that it is less effort to support different brands.

**Reply by cfontanella** [reply]
Ok thanks for your answer and for your work.

**Reply by stritti** [reply]
Hi @tillsteinbach, I saw there some successful messages in the linked issue. Is there something planned to integrate it in VWsFriend? would like to attach my Cupra Born too :)
